Checks/PWR003/solution.f90

```diff
@@ -0,0 +1,26 @@
+! PWR003: Explicitly declare pure functions
+
+! Depends only on its arguments
+! No side effects
+pure function gravity_pure()  
+  real :: gravity
+  gravity = 9.8 ! m/s²
+  return
+end function gravity_pure
```

Comment:
```suggestion
! Returns the gravity on Earth in m/s2
pure function gravity_pure()
  use iso_c_binding, only : c_double
  implicit none
  real(kind=c_double) :: gravity_pure
  gravity_pure = 9.8
end function gravity_pure
```

Note that your version of the function is not returning `9.8`. This new version may change the results of the microbenchmark.

Reply:
My fault

Reply:
Don't forget to update your results after you apply this change.